pages/api/v1/answers.ts

```diff
@@ -290,9 +290,11 @@ export default async function handler(req: NextRequest) {
     const activeActions = actionsWithTags!
       .map((tag) => {
         const currentHost =
-          req.headers.get("x-forwarded-proto") +
-          "://" +
-          req.headers.get("host");
+          req.headers.get("x-forwarded-proto") ??
+          // fallback is safe according to nextjs
+          // (github.com/vercel/next.js/issues/2469#issuecomment-313194091)
+          "http" + "://" + req.headers.get("host");
```

Comment:
I'm a WET man, not WEFT (write everything four times) - I reckon this should be a function :stuck_out_tongue_winking_eye: